cybox/test/__init__.py

```diff
@@ -14,6 +17,89 @@
 logger = logging.getLogger(__name__)
 
 
+@contextlib.contextmanager
+def isolation():
+    """Context manager that attempts to reset the sys.modules and globals()
+    to its initial, pre-`import cybox` state.
+
+    This helps eliminate issues where tests pass because of residuals caused
+    by previous imports.
+
+    Any code which leverages this context manager will need to re-import all
+    dependencies within.
+    """
+    def del_sys_modules(to_del):
+        for mod in to_del:
+            del sys.modules[mod]
+
+    def del_globals(to_del):
+        for g in globals_to_del:
```

Comment:
Haha, yep! It's actually kinda dumb luck that it even works at all. Thanks Python scoping rules!